### PR TITLE
adds license to repo

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview
Closes https://github.com/celestiaorg/orchestrator-relayer/issues/5

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Review by evan-forbes — APPROVED** [review]

**Review by rahulghangas — APPROVED** [review]

**Comment by evan-forbes** [comment]
apparently we can't merge this until we get a codeowner's review lol

@SweeXordious can you force merge still? or do we have to have an admin?

**Comment by SweeXordious** [comment]
I caaaan 
i-am-admin-5b1f54

**Pull request merged**
